tests/integration/test_image_filters.py

```diff
@@ -612,6 +612,37 @@ def test_rag_threshold_filter_on_rgb_image(pil_image, expected_image):
     )
 
 
+def _create_rag_mask(pil_image):
+    mask = np.ones((pil_image.size[1], pil_image.size[0]))
+    mask[:100, :] = 0
+    mask[:, :100] = 0
+    mask[-100:, :] = 0
+    mask[:, -100:] = 0
+    return mask
```

Comment:
I'd move this in the test function line 640. If we will use it in other places we should move it on top of the file right after the imports